custom/enikshay/integrations/bets/const.py

```diff
@@ -1,3 +1,20 @@
+# BETS event names
+VOUCHER_EVENT_ID = "VOUCHER_EVENT_ID"  # TODO: Get this from BETS
+TREATMENT_180_EVENT = "104"
+DRUG_REFILL_EVENT = "109"
+SUCCESSFUL_TREATMENT_EVENT = "SUCCESSFUL_TREATMENT_EVENT"  # TODO: Get this from BETS
+DIAGNOSIS_AND_NOTIFICATION_EVENT = "103"
+AYUSH_REFERRAL_EVENT = "AYUSH_REFERRAL_EVENT"  # TODO: Get this from BETS
```

Comment:
`voucher_type` sounds appropriate - the [listed options](https://docs.google.com/spreadsheets/d/1MCG205FOcsYsmKXHoSTjZ6A1iuqCIAESyleBl7G7PR8/edit#gid=0) are "prescription" and "test" which sound like they'd map cleanly to "chemist" and "lab".  I'd want to confirm that with Clayton or Farid first though.